front/app/components/FormBuilder/components/FormBuilderSettings/LinearAndRatingSettings/ScaleLabelsInput.tsx

```diff
@@ -11,13 +11,10 @@ import { useFormContext } from 'react-hook-form';
 import styled from 'styled-components';
 import { SupportedLocale } from 'typings';
 
-import { ICustomFieldInputType } from 'api/custom_fields/types';
-
 import { useIntl } from 'utils/cl-intl';
```

Comment:
This removes some previous code added in an earlier PR, but it's not needed anymore since we'll use the same labels regardless of the Input Type now.